tests/ui/no_std/no-std-no-start-binary.stderr

```diff
@@ -0,0 +1,6 @@
+error: no Rust entrypoint found
```

Comment:
I think this could be confusing as "clearly" a rust entrypoint exists (`main`). Maybe say that `fn main()` requires libstd?